test/__snapshots__/mathml-spec.js.snap

```diff
@@ -142,7 +141,7 @@ exports[`A MathML builder should output \\limsup_{x \\rightarrow \\infty} correc
       <msub>
         <mo>
           <mi mathvariant="normal">
-            lim sup
+            lim&amp;ThinSpace;sup
```

Comment:
This extra escaping is a bug, caused by `operatorname`'s `mathmlBuilder` creating a `mathMLTree.TextNode`, which escapes its text.  I'm not quite sure how to fix it, other than switching to the Unicode representations...

Reply:
We're definitely doing something wrong here.  In `toNode` we call `document.createTextNode` which expects to be passed an escaped string, yet we're passing `this.text`.  I feel like we should maybe be doing the escaping within the constructor and then we could add an `escape` as an optional second parameter with the default being `true`.